packages/block-library/src/heading/edit.js

```diff
@@ -56,7 +56,7 @@ function HeadingEdit( {
 			</InspectorControls>
 			{ InspectorControlsColorPanel }
 			<TextColor>
-				<BackgroundColorDetector querySelector='[contenteditable="true"]' />
+				<ColorDetector querySelector='[contenteditable="true"]' />
```

Comment:
In the button block, the background and color elements are not the same. Would it make sense to have a colorSelector and backgroundSelector? To allow the detector to work when background and color use different elements?

Reply:
Yes, I was just about to ask you that :smile:

Reply:
Done!